org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/editor/quickfix/XtextGrammarQuickfixProvider.java

```diff
@@ -171,7 +173,24 @@ public void fixEmptyEnumLiteral(final Issue issue, IssueResolutionAcceptor accep
 					enumLiteralDeclaration.setLiteral(keyword);
 				});
 	}
-
+	
+	@Fix(SPACES_IN_KEYWORD)
+	public void fixKeywordNoSpaces(final Issue issue, IssueResolutionAcceptor acceptor) {
+		acceptor.accept(issue, "Split keyword with spaces", "Split keyword with spaces", NULL_QUICKFIX_IMAGE,
+				(ISemanticModification) (EObject element, IModificationContext context) -> {
```

Comment:
I don't think this is a semantic modification. The change is purely based on the document.